### ISSUE TITLE
Can anybody help me theme these last two parts of Yoroi?

### ISSUE THREAD

**Issue by galaxy-nomad** [Emurgo/yoroi-frontend#2312]
Hi please see the attached image. I've scoured the code but can't find the values to change for these last two elements.
![Screen Shot 2021-08-17 at 10 57 57 PM](https://user-images.githubusercontent.com/12664589/129845976-4165c65f-3d67-4f33-93a0-01455984429b.png)

**Comment by galaxy-nomad** [comment]
So basically I can fix the white boxes if the Yoroi developers will change the css from this:

`.UpcomingRewards_card {
    padding: 20px 20px;
    flex: 1;
    border-radius: 6px;
    background-color: #f8f8f8;
    min-width: 234px;
    max-width: 278px;
    margin-bottom: 10px;
}`

To this:

`.UpcomingRewards_card {
    padding: 20px 20px;
    flex: 1;
    border-radius: 6px;
    background-color: var(--theme-nav-item-background-color);
    min-width: 234px;
    max-width: 278px;
    margin-bottom: 10px;
}`

Basically there should be a var value, existing or specific to this item, so that the user can customize it.

**Comment by galaxy-nomad** [comment]
Regarding the graph bars I can't figure out how a custom color would be implemented within the framework of the existing code. Hopefully someone will see this and can help.

**Comment by galaxy-nomad** [comment]
I also need vars for the colors in these items

`.TransferTypeSelect_component .TransferTypeSelect_instructions span a {
    text-decoration: none;
    color: #3154CB;
}

.TransferTypeSelect_component .TransferTypeSelect_instructions span {
    color: #6B7384;
}

.TransferCards_heroCardsItem .TransferCards_heroCardsItemDate {
    color: #6B7384;
    font-size: 12px;
    line-height: 20px;
}

.HorizontalLine_fullWidth {
    border: 0;
    border-bottom: 2px solid #DCE0E9;
    width: 100%;
    margin: 0;
    background-color: #f0f3f5;
    padding: 20px 0 0 0;
}`

**Comment by jarstorm** [comment]
Hello, and thank you for your contribution.
We're currently reviewing long-standing issues to keep the tracker manageable. As this issue hasn't seen activity in some time, we're closing it.
If the problem persists or you'd like to follow up, feel free to comment and we'll reopen it.
